### ISSUE TITLE
Permit tsh login from a list of allowed client IPs

### ISSUE THREAD

**Issue by Richard-Barrett** [gravitational/teleport#11238]
### What
Teleport should be able to limit the IP range from which users can login via the `tsh client`

### How
Configuration context with the teleport Auth Server

### Why
Customer Ask

### Workaround
DNE

gz#4398

**Comment by stevenGravy** [comment]
same/similar to #10201

**Comment by pschisa** [comment]
#10201 would work better as confirmed that the request also would be to include the Teleport Web UI in the IP whitelist since the Web UI provides SSH access as well.  The goal is to force all users to connect to Teleport only after connecting to the VPN

**Comment by zmb3** [comment]
Partially addressed by #12345.

**Comment by reedloden** [comment]
> Partially addressed by #12345.

This ended up being https://github.com/gravitational/teleport/pull/13243.

**Comment by pschisa** [comment]
similar to https://github.com/gravitational/teleport/issues/5971

**Comment by zmb3** [comment]
Closing in favor of #10201
